File: clients/Android/app/src/main/java/org/mtuosc/techchat/models/Board.java
package org.mtuosc.techchat.models;

/**
 * Created by ryan on 11/28/17.
 * Model version of boards
 */

public class Board {
    private String id;
    private String title;
    private String description;

    public Board(String id, String title, String description) {
        this.id = id;
        this.title = title;
        this.description = description;
    }

    public String getId() {
        return id;
    }

    public String getTitle() {
        return title;
    }

    public String getDescription() {
        return description;
    }
}

File: clients/iOS/TechChat/TechChat/models/Board.swift
//
//  Board.swift
//  TechChat
//
//  Created by Ryan Philipps on 11/27/18.
//  Copyright © 2018 OpenSourceClub. All rights reserved.
//

import Foundation


class Board {
    var title: String
    var description: String
    var id: Int
    
    init(id: Int, title:String, description:String) {
        self.id = id
        self.title = title
        self.description = description
    }
}
